src/rust/tests/http_read.rs

```diff
@@ -1,84 +1,88 @@
-use flatgeobuf::*;
-use geozero::error::Result;
-use tokio::runtime::Runtime;
+#[cfg(feature = "http")]
+mod http {
```

Comment:
Just a style nit, but you could hoist this condition to the outer `mod http_read` line.

Reply:
Since this under `tests`, there isn't an explicit `mod http_read` line. If there is a more elegant way to disable tests with feature flags, I'm interested!

Reply:
oh my mistake! I don't know of another way.